internal/ethapi/transaction_args.go

```diff
@@ -149,12 +149,6 @@ func (args *TransactionArgs) setDefaults(ctx context.Context, b Backend) error {
 	if args.To == nil && len(args.data()) == 0 {
 		return errors.New(`contract creation without any data provided`)
 	}
-	if args.GatewayFeeRecipient == nil && !args.EthCompatible {
```

Comment:
Same question from the web3 file... Do we want this version to keep it's behaviour until the fork?

Reply:
Done